viz/MLSMapVisualization.hpp

```diff
@@ -60,6 +60,7 @@ namespace vizkit3d
         Q_PROPERTY(QColor vertical_cell_color READ getVerticalCellColor WRITE setVerticalCellColor)
         Q_PROPERTY(QColor negative_cell_color READ getNegativeCellColor WRITE setNegativeCellColor)
         Q_PROPERTY(QColor uncertainty_color READ getUncertaintyColor WRITE setUncertaintyColor)
+        Q_PROPERTY(bool connected_surface READ isConnectedSurface WRITE setConnectedSurface)
```

Comment:
I would move this flag a bit up, above the QColor properties, otherwise it will be hard to see.

Reply:
True